webapp/src/actions.js

```diff
@@ -1,9 +1,23 @@
 import {getConfig} from 'mattermost-redux/selectors/entities/general';
 
 import {id as pluginId} from './manifest';
-import {STATUS_CHANGE, OPEN_ROOT_MODAL, CLOSE_ROOT_MODAL} from './action_types';
+import {STATUS_CHANGE, OPEN_ROOT_MODAL, CLOSE_ROOT_MODAL, SUBMENU} from './action_types';
 
-export const openRootModal = () => (dispatch) => {
+export const openRootModal = (subMenuText = '') => (dispatch) => {
+    dispatch({
+        type: SUBMENU,
+        subMenu: subMenuText,
+    });
+    dispatch({
+        type: OPEN_ROOT_MODAL,
+    });
+};
+
+export const openRootModalWithSubmenuName = (text) => (dispatch) => {
```

Comment:
Now that we have `export const openRootModal = (subMenuText = '') => (dispatch)`,  `openRootModalWithSubmenuName` is no longer needed ?

Reply:
Apologies for not having removed that.  Done